extensions/adapters/raster/src/main/java/mil/nga/giat/geowave/adapter/raster/RasterUtils.java

```diff
@@ -934,30 +934,37 @@ else if (sourceIsFloat) {
 		 * values. Even if the current image doesn't have NaN values, it could
 		 * have NaN later if the image uses a writable raster.
 		 */
-		NumberRange sourceRange = TypeMap.getRange(sourceType);
 		for (int b = 0; b < numBands; b++) {
-			if (needScaling) {
-				sourceRange = NumberRange.create(
-						min[b],
-						max[b]).castTo(
-						sourceRange.getElementClass());
-				categories[0] = new Category(
-						name[b],
-						null,
-						targetRange,
-						sourceRange);
-			}
-			else {
-				categories[0] = new Category(
-						name[b],
-						null,
-						targetRange,
-						LinearTransform1D.IDENTITY);
-			}
+			// if (needScaling) {
+			// sourceRange = NumberRange.create(
+			// min[b],
+			// max[b]).castTo(
+			// sourceRange.getElementClass());
+			// categories[0] = new Category(
+			// name[b],
+			// null,
+			// targetRange,
+			// sourceRange);
+			// }
+			// else {
+			// categories[0] = new Category(
+			// name[b],
+			// null,
+			// targetRange,
+			// LinearTransform1D.IDENTITY);
+			// }
+			// dst[b] = new GridSampleDimension(
+			// name[b],
+			// categories,
+			// null);
```

Comment:
Should probably either rip this commented code out entirely, OR put a FIXME in here with some notes about why it's commented out so we remember why/when to put it back eventually...